parts/kubernetesmastercustomscript.sh

```diff
@@ -92,6 +93,43 @@ function ensureKubectl() {
     fi
 }
 
+function setNetworkPlugin () {
+    sed -i "s/^KUBELET_NETWORK_PLUGIN=.*/KUBELET_NETWORK_PLUGIN=${1}/" /etc/default/kubelet
+}
+
+function setDockerOpts () {
+    sed -i "s#^DOCKER_OPTS=.*#DOCKER_OPTS=${1}#" /etc/default/kubelet
+}
+
+function configNetworkPolicy() {
+    if [[ ! -z "${APISERVER_PRIVATE_KEY}" ]]; then
+        # on masters
+        ADDONS="calico-configmap.yaml calico-daemonset.yaml"
```

Comment:
If I want to keep the delete then I need it there. (I could move the URL though)